.editorconfig

```diff
@@ -0,0 +1,50 @@
+root = true
+
+[*]
+
+# Indentation and spacing
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+charset = utf-8
+max_line_length = 140
```

Comment:
The only thing I would highly recommend enforcing (and reformat the code afterwards) is the maximum line length. Even on a large screen, you have to constantly scroll horizontally to understand the method signature, which is slightly inconvenient and increases the time for onboarding new contributors.
140 symbols keep a good balance for readability, even on mid-size laptops.

Reply:
I agree, there are a lot of long lines which should be shorter. I think 140 characters is pretty short though, I think it'd be better for my sanity if it was say 180.

Reply:
Based on past project experience, the 140-character line length was recommended as a reasonable limit for code because it strikes a balance between readability and the ability to accommodate different screen sizes. For popular laptops like 15"-16" in a typical workspace, when the solution explorer is opened, there are nearly 140 characters available in a line. Large desktops, of course, are out of this discussion.